packages/docs/docs/reports/index.md

```
@@ -1,28 +1,44 @@
+- [Crossover Point report](/docs/experimental/crossover-point-report)
+- [Budget Analysis report](/docs/experimental/budget-analysis-report)
 
 ## Cash Flow Graph
 
 ![](/img/reports/cash-flow.webp)
 
 Cash flow shows your spending over time. It only looks at budgeted accounts and shows their balance over time, and includes separate income and expense visualizations. It's a quick way to see how your income and expenses affected your available money over time. Your budgeted accounts are considered to be "cash on hand," so this gives you a picture of how available money fluctuates.
 
+## Net Worth Graph
+
+![](/img/reports/net-worth.webp)
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)